### ISSUE TITLE
Discord Server to discuss this plugin in.

### ISSUE THREAD

**Issue by ZepsiZola** [JT122406/HungerGames#29]
**Is your feature request related to a problem? Please describe:**

This plugin is a widely used plugin. It deserves a place for everyone who uses it to go to to seek support with it. When I have an issue with this plugin, my only option is to make an issue on this github or talk about it on spigot.org and neither of those are as quick and easy as a discord server.

**Comment by JT122406** [comment]
I had one but it wasn't getting used enough so I closed it and removed all references to it

**Comment by sovtec** [comment]
> I had one but it wasn't getting used enough so I closed it and removed all references to it

You should open it again ;)

**Comment by JT122406** [comment]
Like I deleted it

**Comment by ZepsiZola** [comment]
Why dont you just start a new one? This plugin is used by a lot of people and so I would anticipate that a discord server would be pretty useful.

**Comment by Yomamaeatstoes** [comment]
I agree you should make a discord server for people who want support and stuff.

**Comment by JT122406** [comment]
So the issue was my server was unused by most people and I started running out of time due to school and life so I wasn't able to provide support and as I was the only one working on it, it became a pain to deal with I am considering it but I am also debating a rewrite and I feel like I should wait till I finish that

**Comment by sovtec** [comment]
> So the issue was my server was unused by most people and I started running out of time due to school and life so I wasn't able to provide support and as I was the only one working on it, it became a pain to deal with I am considering it but I am also debating a rewrite and I feel like I should wait till I finish that

Hello, we have been using HG in some of our events and I enjoy it very mutch. 

If it's something you are interested in, I can create a discord for this plugin and also moderate it for you if you are conserned that will take time from doing what you really want to focus on. I am support in multiple larger discords for plugins like, Husk, UltraColor and Plan, I own 4 servers myself and also run custom moderation bots on all servers. Contakt me on discord if you want to talk more about this. (98%)#6666

**Comment by ZepsiZola** [comment]
> 

You can just start the discord server and have a #support channel but also mention that no one is entitled to get support from you.
